program/tests/close_lookup_table_ix.rs

```diff
@@ -21,10 +20,9 @@ mod common;
 
 #[tokio::test]
 async fn test_close_lookup_table() {
+    // Succesfully close a deactived lookup table.
     let mut context = setup_test_context().await;
-    context
-        .warp_to_slot(SlotHashes::size_of() as u64 + 1)
-        .unwrap();
+    context.warp_to_slot(MAX_ENTRIES as u64 + 1).unwrap();
```

Comment:
The success test is warping to `513`, and the failure test is "warping" to `511`, so let's have this test right at `512` since that's where the program implementation will allow it (assuming I've read the code correctly)

Reply:
Ahh correct, updated!